xdocs/usermanual/component_reference.xml

```diff
@@ -1158,7 +1158,7 @@ props.put("PROP1","1234");</source>
 </note>
 <properties>
     <property name="Name" required="No">Descriptive name for this sampler that is shown in the tree.</property>
-    <property name="Scripting Language" required="Yes">Name of the JSR223 scripting language to be used.
+    <property name="Scripting Language" required="No">Name of the JSR223 scripting language to be used. Use groovy as default. 
```

Comment:
What does `Use groovy as default` mean?

Reply:
@vlsi groovy is the default scripting language